### PR TITLE
issue-4 expand search matching

### PR DESCRIPTION
**Pull request by srveale**

### CODE REVIEW

File: src/screens/posts/PiecesScreen.js [hunk 1]
```diff
@@ -132,9 +132,12 @@ class PiecesScreen extends Component {
...
 								}
 							})
-							.filter(piece =>
-								piece.title.includes(this.state.inputs.searchInput)
-							)
+							.filter(piece => {
+								const searchInput = this.state.inputs.searchInput.toLowerCase();
+								const titleMatch = piece.title.toLowerCase().includes(searchInput);
+								const artistMatch = piece.artist.toLowerCase().includes(searchInput);
+								return artistMatch || titleMatch;
```
Comment (colebowl) [inline src/screens/posts/PiecesScreen.js:139]: Totally being nit-picky here but is there a reason that a match on artist should be returned over title match? Is there  a reason for this to have priority?

Comment (srveale) [reply src/screens/posts/PiecesScreen.js:139]: @colebowl It's not about priority - the piece is included if either artist or title gets matched. The order could be switched and we'd get the same results.

Comment (colebowl) [reply src/screens/posts/PiecesScreen.js:139]: @srveale - totally makes sense! Thanks

**Review by colebowl — APPROVED** [review]
Good to go

**Pull request merged**